docker-compose.yml

```
@@ -21,4 +21,13 @@
     ports:
       - "27017:27017"
     volumes:
-      - ./mongo_data:/data/db
+      - ./mongo_data:/data/db
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)